Solution

Romanskyy opened the pull request.

Romanskyy committed:
Solution

Signed-off-by: Romanskyy Andrey <andrey.romanskyy@icloud.com>

NazarSheremet commented on app/main.py:
```diff
@@ -1,13 +1,30 @@
-    # write your code here
-    pass
+    return [
+        {
+            "errors": [
+                format_linter_error(error)
+                for error in errors if len(errors) > 0
+            ],
+            "path": file_path,
+            "status": "failed" if len(errors) > 0 else "passed",
+        }
```
Use format_single_linter_file function which does the same

Romanskyy replied:
done

NazarSheremet commented on app/main.py:
```diff
@@ -1,13 +1,30 @@
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed",
```
Need to add a check of status, if the status is passed or failed

Romanskyy replied:
done

NazarSheremet reviewed (changes requested):
Need some changes

Romanskyy committed:
Making review's corrections

- used format_single_linter_file function inside format_linter_report func
- add a check of a status inside format_single_linter_file func

Signed-off-by: Romanskyy Andrey <andrey.romanskyy@icloud.com>

Romanskyy commented:
please, review my PR

NazarSheremet reviewed (approved):
Good job!

YuriiSmolii reviewed (approved):
LGTM
